app/javascript/textarea-markdown-linkify.js

```diff
@@ -0,0 +1,36 @@
+import MarkdownIt from 'markdown-it'
+
+export default class {
+  linkify(selector) {
+    const textareas = document.querySelectorAll(selector)
+    if (textareas.length === 0) {
+      return null
+    }
+
+    Array.from(textareas).forEach((textarea) => {
+      textarea.addEventListener('paste', async (event) => {
+        event.preventDefault()
+        const { selectionStart, selectionEnd } = textarea
+        const selectedText = textarea.value.slice(selectionStart, selectionEnd)
+        // headless chromeではevent.clipboardData.getData('text')は空文字を返すため、代わりにnavigator.clipboard.readText()を使用
+        // https://github.com/fjordllc/bootcamp/pull/6747#discussion_r1325362833
+        const pasteText =
+          event.clipboardData.getData('text') ||
+          (await navigator.clipboard.readText())
```

Comment:
好みの問題ですが、2行ぐらいに留めた方が読みやすい気がします。

```suggestion
        const pasteText =
          event.clipboardData.getData('text') || (await navigator.clipboard.readText())
```

Reply:
提示いただいたコードにするとlintで警告が出るため、[724d9bf](https://github.com/fjordllc/bootcamp/pull/6747/commits/724d9bfda674f095c5df633b493b8d8b1b3052d2)で元に戻しました。
(個人的には3行よりも2行の方が読みやすいです)

Reply:
まじかー。

```javascript 
const pasteText = event.clipboardData.getData('text') || (await navigator.clipboard.readText())
```

こんな感じで1行にしてもダメですかね？

lint的にどうしようもないなら今のままで問題ありません〜。

Reply:
1行にしてもダメだったので、今のままで行こうと思います。